src/locale/messages.de.xlf

```diff
@@ -295,7 +295,22 @@
         <target>Die Auswahl wurde erfolgreich übermittelt</target>
         <context-group purpose="location">
           <context context-type="sourcefile">src/app/_services/api-order.service.ts</context>
-          <context context-type="linenumber">83</context>
+          <context context-type="linenumber">82</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="5947760680752260655" datatype="html">
+        <source>Commande passée avec succès! Vous recevrez un email lorsque tous les téléchargements seront prêts.</source>
```

Comment:
Did we lose the target (the translation) for this text item? Should be something like "Ihre Bestellung war erfolgreich..."